bpf/headers/maps.h

```diff
@@ -43,6 +44,25 @@ struct bpf_map __section("maps") process_ip = {
     .max_entries = 1024,
     .map_flags = 0,
 };
+#endif
+
+#if ENABLE_IPV6
+struct bpf_map __section("maps") local_pod_ip6s = {
```

Comment:
I think we probably don't need to distinguish between v4 and v6 maps, maybe a union type would work?

Reply:
There are 2 reasons:

1. Reducing memory usage as the comment above
2. I'm not sure if we are gonna support dual stack, if yes, this would be helpful, because there's no way to store ipv4/v6 addr of the same pod with a union type

Reply:
It's for saving memory as the comment above

Reply:
Yeah that's a good point, will try it